streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -144,7 +145,8 @@ public StreamTask(final TaskId id,
         final DeserializationExceptionHandler defaultDeserializationExceptionHandler = config.defaultDeserializationExceptionHandler();
         for (final TopicPartition partition : partitions) {
             final SourceNode source = topology.source(partition.topic());
-            final TimestampExtractor sourceTimestampExtractor = source.getTimestampExtractor() != null ? source.getTimestampExtractor() : defaultTimestampExtractor;
+            final TimestampExtractor timestampExtractor = source.getTimestampExtractor();
```

Comment:
We get the extractor from the source; should the var name not be `sourceTimestampExtractor` ?

Reply:
Sounds good, I can do that.